src/input/pedal.cpp

```
@@ -69,66 +76,131 @@ void Pedal::update() {
+        // Lectura cambió, resetear contador
+        staticReadCount = 0;
+    }
+    lastRaw = raw;
+    
+    // ========================================
+    // VALIDACIÓN 3: Detección de valores extremos sostenidos
+    // ========================================
+    // Si ADC está pegado en 0 o 4095, probablemente hay problema eléctrico
+    static uint8_t extremeValueCount = 0;
+    
+    if (raw <= PedalValidation::ADC_EXTREME_LOW || raw >= PedalValidation::ADC_EXTREME_HIGH) {  // Cerca de límites ADC
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)